app/shared/index.js

```diff
@@ -70,6 +70,7 @@ export { default as AppStateChange } from './src/AppStateChange';
 export { default as ReadMore } from './src/ReadMore';
 export { default as WithStorage } from './src/WithStorage';
 export { default as GeolocationContext } from './src/GeolocationContext';
+export { default as PdfViewer } from './src/PdfViewer';
```

Comment:
Can we maybe create separate PDF package (workspace)? I would like to start splitting the shared package into multiple workspaces because of our problem with circular dependencies.